service/controller/app/resource/releasemigration/create.go

```diff
@@ -115,6 +115,21 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	// It means helm release migration is in progress.
 	if hasConfigMap && hasSecret {
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("release %#q helmV3 migration in progress", key.ReleaseName(cr)))
+
+		found, err := findMigrationApp(ctx, cc.Clients.Helm, tillerNamespace)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		if !found {
```

Comment:
If the operator found out remaining helm V2 releases from clusters, it should re-initiate installing the migration app.